site/kubernetes/operator/kubectl-plugin.md

```diff
@@ -0,0 +1,211 @@
+
+This will create RabbitMQ cluster with some basic configuration where only the cluster name is configured.
+
+### <a id='get-rmq' class='anchor' href='#get-rmq'>Get a RabbitMQ cluster</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq get INSTANCE
+</pre>
+
+Display all of the kubernetes resources associated with the named RabbitMQ cluster including
+pods, configmaps, statefulsets, services and secrets.
+
+<pre class="lang-bash">
+NAME                     READY   STATUS    RESTARTS   AGE
+pod/hello-rmq-server-0   0/1     Pending   0          5h31m
+
+NAME                               DATA   AGE
+configmap/hello-rmq-plugins-conf   1      5h31m
+configmap/hello-rmq-server-conf    2      5h31m
+
+NAME                                READY   AGE
+statefulset.apps/hello-rmq-server   0/1     5h31m
+
+NAME                      TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)              AGE
+service/hello-rmq         ClusterIP   10.100.156.4   none        5672/TCP,15672/TCP   5h31m
+service/hello-rmq-nodes   ClusterIP   None           none        4369/TCP,25672/TCP   5h31m
+
+NAME                             TYPE     DATA   AGE
+secret/hello-rmq-default-user    Opaque   3      5h31m
+secret/hello-rmq-erlang-cookie   Opaque   1      5h31m
+</pre>
+
+### <a id='list-rmq' class='anchor' href='#list-rmq'>List all RabbitMQ clusters</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq list
+</pre>
+
+Displays all RabbitMQ clusters deployed by the RabbitMQ cluster operator on the target kuberentes instance.
+
+### <a id='delete-rmq' class='anchor' href='#delete-plugin'>Delete RabbitMQ cluster</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq delete INSTANCE
+</pre>
+
+Delete a RabbitMQ cluster, or multiple RabbitMQ clusters. When deleting multiple RabbitMQ clusters, provide a space
+seperated list.
+
+<pre class="lang-bash">
+kubectl rabbitmq delete rmq1 rmq2 rmq3
+</pre>
+
+### <a id='secrets' class='anchor' href='#secrets'>Print default user secrets</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq secrets INSTANCE
+</pre>
+
+Displays the default user secrets for the named RabbitMQ cluster
+
+### <a id='manage' class='anchor' href='#manage'>Open the Management UI</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq manage INSTANCE
+</pre>
+
+Opens the RabbitMQ Management dashboard in a browser.
+
+### <a id='debug' class='anchor' href='#debug'>Set Log Level to debug</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq debug INSTANCE
+</pre>
+
+Sets the log level on all nodes to debug. For a detailed breakdown on RabbitMQ logging
+see the [logging docs page](/logging.html).
+
+### <a id='tail' class='anchor' href='#tail'>Tail Logs</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq tail INSTANCE
+</pre>
+
+Attach to live log output from all RabbitMQ nodes.
+
+### <a id='observe' class='anchor' href='#observe'>Observe</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq observe INSTANCE INDEX
+</pre>
+
+Opens the rabbitmq-diagnostics observer interface for a given node. For more
```

Comment:
We can link to the [`rabbitmq-diagnostics(8)` man page](https://www.rabbitmq.com/rabbitmq-diagnostics.8.html) here.

Reply:
Good idea, thanks.